### ISSUE TITLE
loss and acc curves

### ISSUE THREAD

**Issue by andriyrizhiy** [HuiZeng/Grid-Anchor-based-Image-Cropping#14]
Can i get your loss and acc curves during training?

**Comment by HuiZeng** [comment]
Sorry, I did not save the loss and acc curves in the CVPR version. In that version, I almost did not use data augmentation, so the training loss is much lower than the validation loss.

In the extended journal version, I do have the loss curves as follows:
![image](https://user-images.githubusercontent.com/12780780/63351638-2f6fac00-c392-11e9-9224-08de9e832cda.png)
The base model is also changed in the journal version. But the loss cuvres have the same tendancy.

**Comment by andriyrizhiy** [comment]
in your experiment val acc also stood still like val loss curve?

**Comment by andriyrizhiy** [comment]
becouse i get like this 
![image](https://user-images.githubusercontent.com/41432542/63408564-03871180-c3f8-11e9-98a3-9b62a7527ba9.png)
 for acc4/5 (train is green and val is gray)
It is not used ROD (only roi_align)

**Comment by HuiZeng** [comment]
I did not plot the acc curves. But they should have the similar tendency as the loss curves.

**Comment by andriyrizhiy** [comment]
what is end value of you train acc? it is much more than val acc?

**Comment by HuiZeng** [comment]
Yes, the training acc is higher than the val acc.

**Comment by andriyrizhiy** [comment]
but not too mush
